controllers/nodelister_mock.go

```diff
@@ -16,6 +16,7 @@ limitations under the License.
 
 // Code generated by MockGen. DO NOT EDIT.
 // Source: ./azuremanagedmachinepool_reconciler.go
+// TODO: Add this to make generate target. So far it seems impossible to update this using generation.
```

Comment:
Is this TODO intended to be left here? If so, it could help to create an issue so it isn't forgotten.